centos_ci.sh

```diff
@@ -292,14 +292,40 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
```

Comment:
Do we need to be able to specify the repo?

Reply:
All parameters are optional.

So, if I want to test how documentation render for my branch then, I need to specify branch and repo. Otherwise, we need to push our branch to upstream always to test it.

Reply:
But now you can easily get confused about what's getting pushed. One thinks the docs from the master repo are published, but in reality, it is something else.

Reply:
@hferentschik , I think you are confusing with the intent here.

I have created another issue https://github.com/minishift/minishift/issues/1761 which will maintain a published doc on each PR merge.

The purpose of this PR is to give ability to developer to see how published docs will look like from his desired changes.
This is the path I am assuming user will take when he see any documentation task.

- Create a branch (say `issue-docs-new-cmd`)
- Add necessary documentation changes
- Run locally via `make gen_docs; make serve_docs`.

*NOTE*: Currently, there is no way how you can validate how docs change will look when published after release.

With this PR, developer can see how it will look when published
- User will trigger [minishift-job](https://ci.centos.org/job/minishift-docs/) by passing parameters `BRANCH` ( `issue-docs-new-cmd` in this case) and `REPO` ( `https://github.com/<username>/minishift` in this case).
- Once, developer is sure that the changes looking fine in published doc from above job, he can send PR referring this published doc. It will help reviewer to have and verify the docs changes easily.

Not sure how one can easily get confused as you mentioned in above comments. 

I completely agreed this PR need documentation changes.  

@minishift/minishift-dev  Any thoughts / comments / suggestion.

Reply:
>  I think you are confusing with the intent here.

I understood it once I saw where you are publishing to. 

> Currently, there is no way how you can validate how docs change will look when published after release.

You can build the integrated docs as per documentation, right?

> Not sure how one can easily get confused as you mentioned in above comments.

Since I initially thought this all gets pushed to the same directory. If there would have been some docs, I maybe would have known earlier the intend ;-)

Reply:
> You can build the integrated docs as per documentation, right?

Yes, documentation is in place but lot of work. Who else verify the docs change by creating doc tar from someone's PR and serve openshift-docs locally :wink: 

Hence, this PR :) Now, reviewer can just need PR owner repo link and branch and after 10 mins review production like documentation :smile: 

> Since I initially thought this all gets pushed to the same directory. If there would have been some docs, I maybe would have known earlier the intend ;-)

Agreed!  Will take care next time  :+1: